resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -13,7 +13,7 @@
 options.old_el = 0
 options.old_sles = 0
 options.sles = 0
-options.java = 'java-1.8.0-openjdk-headless'
+options.java = 'java-11-openjdk-headless | java-17-openjdk-headless | java-21-openjdk-headless | java-25-openjdk-headless'
```

Comment:
This ends up in the Fedora case and this isn't valid for Fedora because it's the Debian syntax.